tests/draft2019-09/refRemote.json

```diff
@@ -17,7 +17,7 @@
     },
     {
         "description": "fragment within remote ref",
-        "schema": {"$ref": "http://localhost:1234/subSchemas.json#/integer"},
+        "schema": {"$ref": "http://localhost:1234/subSchemas.json#/$defs/integer"},
```

Comment:
I think there's similar churn here too in renaming these -- I don't think this is strictly required by 2019-09, so would be good to keep these kinds of changes out if possible.

Reply:
Same as above: it's required because a conformant implementation isn't required to look at the unknown "integer" keyword.

Reply:
@Julian I'm with @jgraettinger on this, having an unknown keyword here is confusing.  Since it is not a nested unknown keyword I expect it would generally work, but the point of introducing that wording was to clarify what *reliably* works vs what might not.

Reply:
Yeah it was left as-is, we moved the unknown keyword test away and into optional.